### DISCUSSION TITLE
Troubles Receiving WATTS Wireless Analog Room Thermostat BT-DP02-RF with rtl_433

### DISCUSSION THREAD

**Discussion by GHkludo** [merbanan/rtl_433#2712] [General]
Hello, community,

Lately, I've been trying to receive my WATTS Wireless Analog Room Thermostat BT-DP02-RF using rtl_433, but unfortunately, I've been unsuccessful. I've taken all the necessary steps to establish the connection, but it seems like rtl_433 isn't recognizing or interpreting the signal.

Has anyone here successfully attempted to receive this particular room thermostat with rtl_433? If so, could you please share your experiences and insights? There might be specific settings or tricks that I've overlooked. Any help or guidance would be greatly appreciated.

Thank you in advance for your support! I look forward to your feedback and experiences.

Best regards,
GHkludo

**Comment by zuckschwerdt** [comment]
I don't remember adding a decoder for a "WATTS BT-DP02-RF". Can you grab samples and attach here as zip? https://triq.org/rtl_433/ANALYZE.html

**Reply by GHkludo** [reply]
Thank you for the quick response. I've spent a few minutes creating a sample, and I hope it can be useful to you.
[SAMPLE_868.3M_1000k.zip](https://github.com/merbanan/rtl_433/files/13271649/SAMPLE_868.3M_1000k.zip)

**Comment by zuckschwerdt** [comment]
There are multiple senders in that collection, I counted at least ~7. Can you record without antenna to isolate the sender? Then drop samples on https://triq.org/pdv/ and check visually that they look so similar as to be from one device.

**Reply by GHkludo** [reply]
I now have three samples from the same device. I had removed the antenna to be certain.
[SAMPLE#2_868.3M_1000k.zip](https://github.com/merbanan/rtl_433/files/13272508/SAMPLE.2_868.3M_1000k.zip)

**Reply by zuckschwerdt** [reply]
Looks mostly ok, but there is clipping (too "loud") try to increase the distance and check in pdv that the "ghost" above the signal disappears.
The signal seems to be 50 µs PCM GFSK and our demod needs a very clean signal to work with GFSK.
![watts](https://github.com/merbanan/rtl_433/assets/12833/4ab5716f-b8a2-48f7-a4e5-769f89560f75)

**Reply by zuckschwerdt** [reply]
This should basically work `rtl_433 -f 868.3M -Y autolevel -M level -M noise -R 0 -X 'n=name,m=FSK_PCM,s=50,l=50,r=1000'`
For the samples you get:
```
{163}000224142c12082400004005400002008001a4382
{158}0010418a04808500000811d00001005002007280
{175}000013051429408120000001114000040140040400a0
```
Which does not seem right, the start should be `5555` or `aaaa`.

**Reply by GHkludo** [reply]
I wasn't entirely sure what to do, but I ended up creating two samples with different distances. Hopefully, these look better.
[SAMPLE#3_868.3M_1000k.zip](https://github.com/merbanan/rtl_433/files/13284077/SAMPLE.3_868.3M_1000k.zip)

**Reply by zuckschwerdt** [reply]
The second one is not from the same device ;) The first sample looks really good. Sadly the demod can't reliably pick up the data.
Looking sharply at signal the data should be something like `aaaa d391` -- which would perfectly match a known sync-word.

Best you can do is to try random things until a code like that appears. Sorry.
